projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -75,6 +75,17 @@ public function register_routes() {
 				'methods'             => \WP_REST_Server::READABLE,
 				'callback'            => array( $this, 'get_all_integrations_status' ),
 				'permission_callback' => array( $this, 'get_items_permissions_check' ),
+				'args'                => array(
+					'version' => array(
+						'type'              => 'integer',
+						'default'           => 1,
+						'sanitize_callback' => 'absint',
+						'validate_callback' => function ( $param ) {
+							$version = absint( $param );
+							return in_array( $version, array( 1, 2 ), true );
+						},
```

Comment:
Added version param, with default set to 1 for backward compatibility. Our useIntegrationsStatus hook is updated to pass version 2.